nextjs-legacy/README.md

```diff
@@ -0,0 +1,52 @@
+This is an example of an instrumented Next.js app.
+
+## Overview
+
+This example application shows a list of users and provides a form for editing
+individual users. When editing a user's name, no error will occur. When editing
+a user's age, an error condition will be encountered.
+
+A few source files to pay particular attention to:
+
++ [next.config.js](./next.config.js): shows configuration necessary to get
+instrumentation for all of the libraries New Relic's Node.js agent supports.
++ [newrelic.js](./newrelic.js): shows how to enable application logging. This
+application forwards server-side logs via our instrumentation of the Pino logger.
++ [pages/_document.js](./pages/_document.jsx): shows how to enable the browser
+agent.
++ [pages/user/[id].jsx](./pages/user/%5Bid%5D.jsx): shows how to handle errors
```

Comment:
this is handling errors but not surfacing them as errors in New Relic.

Reply:
You'd need to do something like [this](https://github.com/newrelic-experimental/newrelic-nextjs-integration/blob/main/pages/_error.tsx)

Reply:
Can you provide some hints about what that code is doing?

Reply:
I have added the catch-all page.